src/process-effects.js

```diff
@@ -18,6 +18,15 @@ module.exports = tagFunction => {
 	// create the component, which will save new effects to the effect queue
 	const tagResult = tagFunction()
 
+	if (tagResult instanceof Array) {
+		throw new Error('Tram-One: Sorry, Tram-One does not currently support array returns. Wrap components in an element before returning.')
+	}
+
+	// verify that the tagResult is an element (if it's not, we won't be able to run effects or do anything useful)
+	if (!(tagResult instanceof Element)) {
+		throw new Error(`Tram-One: expected component to return an Element, instead got ${typeof tagResult}. Verify the component is a function that returns DOM.`)
+	}
```

Comment:
Error to throw when a component does not return an element.
Previously, this would throw the following error, because no DOM object was being returned.
```
"Cannot set property 'tram-tag-new-effects' of undefined"
```
If the thing being returned was not an element, other issues would also occur (most notably effects would not process).

Now we verify that something is returned, and that it's an Element.